src/components/ForecastDetails.js

```diff
@@ -7,19 +7,58 @@ function ForecastDetails({ forecast }) {
 
   const formattedDate = new Date(date).toDateString();
 
+  let windDirection = "";
+  const { direction } = wind;
+
+  if (direction === "n") {
+    windDirection = "North";
+  } else if (direction === "nne") {
+    windDirection = "North North-East";
+  } else if (direction === "ne") {
+    windDirection = "North East";
+  } else if (direction === "ene") {
+    windDirection = "East North-East";
+  } else if (direction === "e") {
+    windDirection = "East";
+  } else if (direction === "ese") {
+    windDirection = "East South-East";
+  } else if (direction === "se") {
+    windDirection = "South East";
+  } else if (direction === "sse") {
+    windDirection = "South South-East";
+  } else if (direction === "s") {
+    windDirection = "South";
+  } else if (direction === "ssw") {
+    windDirection = "South South-West";
+  } else if (direction === "sw") {
+    windDirection = "South West";
+  } else if (direction === "wsw") {
+    windDirection = "West South-West";
+  } else if (direction === "w") {
+    windDirection = "West";
+  } else if (direction === "wnw") {
+    windDirection = "West North-West";
+  } else if (direction === "nw") {
+    windDirection = "North West";
+  } else {
+    windDirection = "North North-West";
+  }
```

Comment:
While this approach works, it could be made a bit tidier. First you could create an object that pairs the various wind directions with their plain text equivalent:
```
const windMap = {
  n: "North",
  nne: "North North-East",
  ne: "North East",
  ene: "East North-East",
  e: "East",
  ese: "East South-East",
  se: "South East",
  sse: "South South-East",
  s: "South",
  ssw: "South South-West",
  sw: "South West",
  wsw: "West South-West",
  w: "West",
  wnw: "West North-West",
  nw: "North West",
  nnw: "North North-West"
};
```
Then, look up a value in this new object by the key passed in and assign it to the `windDirection` variable:
```
const windDirection = windMap[wind.direction];
```
Halves the amount of lines taken up, and makes it easier to read to boot :)